scripts/utils/exec.ts

```diff
@@ -28,51 +29,48 @@ async function exec(txData, totalValue, addresses) {
     });
   }
 
-  const governor = await ethers.getContractAt('GovernorAlpha', governorAlphaAddress);
+  execProposal(voterAddress, governorAlphaAddress, totalValue, proposalNo);
+}
 
-  const proposalNo = await governor.latestProposalIds(proposerAddress);
+export async function execProposal(voterAddress, governorAlphaAddress, totalValue, proposalNo) {
+  const governor = await ethers.getContractAt('FeiDAO', governorAlphaAddress);
+  const signer = await getImpersonatedSigner(voterAddress);
 
   console.log(`Proposal Number: ${proposalNo}`);
 
   let proposal = await governor.proposals(proposalNo);
   const { startBlock } = proposal;
 
   // Advance to vote start
-  if ((await time.latestBlock()) < startBlock) {
+  if (toBN(await time.latestBlock()) < toBN(startBlock)) {
```

Comment:
if we're using a BN, I wouldn't think they would work with the `<` symbol as it is an object, not a number.

Reply:
It appears that this works. They must have comparators built-in. I can change to bn method

Reply:
You don't have to use the bn built in, you could just cast to a type Number